ods_ci/tests/Resources/Page/ODH/Prometheus/Prometheus.robot

```diff
@@ -335,3 +335,26 @@ Run Query Range
     ...    headers=${pm_headers}    verify=${False}
     Request Should Be Successful
     RETURN    ${resp}
+
+Set Prometheus Variables
+    [Documentation]    Update Global Variables RHODS_PROMETHEUS_URL and RHODS_PROMETHEUS_TOKEN if missing
+    ${result} 	Run Process 	oc get route -l app\=prometheus -A -o jsonpath\='{.items[0].metadata.namespace}'    shell=yes
+    IF    ${result.rc} == 0
+        ${prom_namespace}=    Set Variable    ${result.stdout}
+        Log    Prometheus is running in the namespace: ${prom_namespace}    console=True
```

Comment:
is it assuming that the first instance of prometheus it find in the cluster is the correct one?

Reply:
have you tried it in a OSD/ROSA cluster?

Reply:
On OSD, not on ROSA.

Reply:
hm okay, I'm unsure if ROSA might have any additional prometheus which could break the flow. I guess we will find out